docs/tutorials/average_optimizers_callback.ipynb

```diff
@@ -76,9 +76,9 @@
         "\n",
```

Comment:
<p>I think it would be worth adding a line or two explain how one needs to be careful about setting up the checkpoint paths. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/2497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I don't think thats neccesary because its only expected from the user and doesn't relate heavily to our main objective here. Thats my opinion atleast.</p>

Reply:
I referred to [your comment](https://github.com/tensorflow/addons/pull/2497#issuecomment-855204867):

> The notebook was using the same folder to store the weights of for each optimizer. This lead to loading the same weights for each optimizer.

Reply:
Yes, that was an error I fixed.

Reply:
Yeah, so chances are that users won't notice this. So, explicitly making it clear inside the tutorial is helpful.